src/puppetlabs/trapperkeeper/authorization/ring_middleware.clj

```diff
@@ -17,6 +18,11 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;; Private
 
+(def OIDMap
+  "Mapping of string OIDs to shortname keywords. Used to update an incoming
+  request with a shortname -> value extensions map."
+  {String schema/Keyword})
```

Comment:
Better to use `schema/Str` than `String` here?  Not sure we've always been consistent about that anyway...